python/requirements.txt

```diff
@@ -43,6 +43,9 @@ starlette
 
 # Requirements for running tests
 blist; platform_system != "Windows"
+azure-common
+azure-mgmt-resource
+msrestazure
```

Comment:
@richardliaw , what do you think about adding 46-48 to the requirements?

Reply:
this is only for tests right? i think this would be OK